packages/x/xviewer-plugins/package.yml

```diff
@@ -0,0 +1,103 @@
+name       : xviewer-plugins
+version    : 3.4.0
+release    : 1
+source     :
+    - https://github.com/linuxmint/xviewer-plugins/archive/refs/tags/3.4.0.tar.gz : c9427577396bc19b0d7fa1cc8b69693973e19cd461ec41cdb2e60f638c1b513f
+homepage   : https://github.com/linuxmint/xviewer-plugins
+license    : GPL-2.0-or-later
+component  : desktop
+summary    :
+    - Plugins for xviewer
+    - common : Common data required for plugins
+    - ^xviewer-plugin-exif-display : xviewer plugin to display camera settings and histogram
+    - ^xviewer-plugin-export-to-folder : xviewer plugin to export the current image to a separate directory
+    - ^xviewer-plugin-map : xviewer plugin to display the geolocation of the image on a map
+    - ^xviewer-plugin-postasa : xviewer to upload your pictures to PicasaWeb
+    - ^xviewer-plugin-pythonconsole : Python console for xviewer
+    - ^xviewer-plugin-send-by-mail : xviewer plugin to send an image attached to a new mail
+    - ^xviewer-plugin-slideshowshuffle : xviewer plugin to shuffle images in slideshow mode
+description:
+    - Plugins for xviewer.
+    - common : Common data required for plugins.
+    - ^xviewer-plugin-exif-display : xviewer plugin to display camera settings and histogram.
+    - ^xviewer-plugin-export-to-folder : xviewer plugin to export the current image to a separate directory.
+    - ^xviewer-plugin-map : xviewer plugin to display the geolocation of the image on a map.
+    - ^xviewer-plugin-postasa : xviewer to upload your pictures to PicasaWeb.
+    - ^xviewer-plugin-pythonconsole : Python console for xviewer.
+    - ^xviewer-plugin-send-by-mail : xviewer plugin to send an image attached to a new mail.
+    - ^xviewer-plugin-slideshowshuffle : xviewer plugin to shuffle images in slideshow mode.
+builddeps  :
+    - pkgconfig(champlain-0.12)
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(libgdata)
+    - pkgconfig(libgnomekbdui)
+    - pkgconfig(xkbfile)
+    - pkgconfig(xviewer)
+setup      : |
+    %meson_configure
+build      : |
+    %ninja_build
+rundeps    :
```

Comment:
Any reason to put rundeps between build and install phase?

Reply:
I have no idea how I managed to do that, lol. Will fix later today.